src/react/components/pages/editorPage/canvas.test.tsx

```diff
@@ -58,7 +58,8 @@ describe("Editor Canvas", () => {
         const editorMock = Editor as any;
         editorMock.prototype.addContentSource = jest.fn(() => Promise.resolve());
         editorMock.prototype.scaleRegionToSourceSize = jest.fn((regionData: any) => regionData);
-        editorMock.prototype.RM = new RegionsManager(null, null, null);
+        editorMock.prototype.RM = new RegionsManager(null, null);
```

Comment:
not sure if this from a library or our code, but using abbreviations like these, `RM` & `AS`, makes it really hard to read the code

Reply:
This is from the canvastools library that we are using